src/cpp/include/openvino/genai/generation_config.hpp

```diff
@@ -66,6 +70,7 @@ class OPENVINO_GENAI_EXPORTS GenerationConfig {
     size_t max_new_tokens = SIZE_MAX;
     size_t max_length = SIZE_MAX;
     bool ignore_eos = false;
+    size_t min_new_tokens = 0;
```

Comment:
Looks like support of `num_return_sequence` is lost. Please do not forget about this parameter